### PR TITLE
I885: Fixed typo (Utilites->Utilities) in file names and class usages

### LINKED ISSUE #885: core/execute/JudgementUtilites.java  typo in Utilities
Typographical errors in class names:   `JudgementUtilites` should be `JudgementUtilities` (or `JudgmentUtilities`, according to some people);  `JudgementUtilitesTest` should be `JudgementUtilitiesTest`.

### PR DESCRIPTION
**Pull request by zilong-yang**
### Description of what the PR does

Fixed typos in class name `JudgementUtilites` and `JudgementUtilitesTest` and their usages in other classes.

### Issue which the PR addresses
Fixes #885 

### Environment in which the PR was developed (OS,IDE, Java version, etc.)
MacOS Ventura
Jetbrains IntelliJ
Java 1.8 (Amazon Corretto)

### Precise steps for _testing_ the PR (i.e., how to demonstrate that it works correctly)
1. Run unit tests in _JudgementUtilitiesTest.java_ and make sure all tests still pass

### CODE REVIEW

File: src/edu/csus/ecs/pc2/services/web/EventFeedStreamer.java
Comment (johnbrvc) [inline src/edu/csus/ecs/pc2/services/web/EventFeedStreamer.java:1]: Copyright date should be updated.

File: src/edu/csus/ecs/pc2/ui/EditRunPane.java
Comment (johnbrvc) [inline src/edu/csus/ecs/pc2/ui/EditRunPane.java:1]: Copyright should be updated.

File: test/edu/csus/ecs/pc2/core/execute/JudgementUtilitiesTest.java
Comment (johnbrvc) [inline test/edu/csus/ecs/pc2/core/execute/JudgementUtilitiesTest.java:1]: Copyright should be updated.

**Review by johnbrvc — CHANGES_REQUESTED** [review]
Loaded and build and ran tests Ok (why shouldn't it?)
Just the copyright dates that should be updated.

File: src/edu/csus/ecs/pc2/services/web/EventFeedStreamer.java
Comment (zilong-yang) [reply src/edu/csus/ecs/pc2/services/web/EventFeedStreamer.java:1]: done!

File: src/edu/csus/ecs/pc2/ui/EditRunPane.java
Comment (zilong-yang) [reply src/edu/csus/ecs/pc2/ui/EditRunPane.java:1]: done!

File: test/edu/csus/ecs/pc2/core/execute/JudgementUtilitiesTest.java
Comment (zilong-yang) [reply test/edu/csus/ecs/pc2/core/execute/JudgementUtilitiesTest.java:1]: done!

**Review by johnbrvc — APPROVED** [review]

File: src/edu/csus/ecs/pc2/services/web/EventFeedStreamer.java
Comment (johnbrvc) [reply src/edu/csus/ecs/pc2/services/web/EventFeedStreamer.java:1]: For future reference, only the CREATOR of a comment can resolve it for PC2.

File: src/edu/csus/ecs/pc2/ui/EditRunPane.java
Comment (johnbrvc) [reply src/edu/csus/ecs/pc2/ui/EditRunPane.java:1]: For future reference, only the CREATOR of a comment can resolve it for PC2.

File: test/edu/csus/ecs/pc2/core/execute/JudgementUtilitiesTest.java
Comment (johnbrvc) [reply test/edu/csus/ecs/pc2/core/execute/JudgementUtilitiesTest.java:1]: For future reference, only the CREATOR of a comment can resolve it for PC2.

**Review by clevengr — APPROVED** [review]
Looks good; adding an approval.  I'll also merge the PR.

**Pull request merged**